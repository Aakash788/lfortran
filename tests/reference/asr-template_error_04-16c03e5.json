{
    "basename": "asr-template_error_04-16c03e5",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/template_error_04.f90",
    "infile_hash": "9e6c88e23b84ebd6b1f8ca22a0e33abe3cb9bc158bdcc30a5526c4c7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-template_error_04-16c03e5.stderr",
    "stderr_hash": "95c5e00efc586866b197255ae870696d01ee5c9d18b5f6f1fdff0fda",
    "returncode": 2
}