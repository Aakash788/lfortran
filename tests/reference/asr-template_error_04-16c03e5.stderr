syntax error: Token 'requires' (of type 'identifier') is unexpected here
  --> tests/errors/template_error_04.f90:15:9
   |
15 |         requires R(T, F)
   |         ^^^^^^^^ 
